js/ssai_pulse.js

```diff
@@ -776,6 +777,7 @@ OO.Ads.manager(function(_, $)
       var mainUrl = urlParts[0];
       var mainUrlParts = mainUrl.split("/");
       if (mainUrlParts !== null) {
+      	this.domainName = mainUrlParts[2];
```

Comment:
Can we update the jsdoc to include that this function also parses for the domain name?

Reply:
@aeng7 added